README.md

```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
 
-### Run the [score_credit.py](test/score_credit.py) script following the  [test/README.md](test/README.md) instructions
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](../../README.md#create-a-watson-openscale-service)
 
-Output:
+</details>
 
-```bash
-******************************************
-Prepare scoring payload ...
-Score the model ...
-Return predictions ...
+### 3. Create a Watson OpenScale service
 
-{'fields': ['prediction', 'probability'], 'labels': ['Risk', 'No Risk'], 'values': [['No Risk', [0.8823126094462725, 0.1176873905537274]], ['No Risk', [0.6755090846150376, 0.3244909153849625]], ['No Risk', [0.8944991421537971, 0.10550085784620292]], ['No Risk', [0.9297263621482206, 0.07027363785177945]]]}
+<details><summary>Setup OpenScale on Cloud Pak for Data if needed</summary>
 
-******************************************
-```
+> Note: This assumes that your Cloud Pak for Data Cluster Admin has already installed and provisioned OpenScale on the cluster.
+
+* In the Cloud Pak for Data instance, go the (☰) menu and under `Services` section, click on the `Instances` menu option.
+
+  ![Service](doc/source/images/services.png)
+
+* Find the `OpenScale-default` instance from the instances table and click the three vertical dots to open the action menu, then click on the `Open` option.
+
+  ![Openscale Tile](doc/source/images/services-wos-instance.png)
+
+* If you need to give other users access to the OpenScale instance, go the (☰) menu and under `Services` section, click on the `Instances` menu option.
+
+* Find the `OpenScale-default` instance from the instances table and click the three vertical dots to open the action menu, then click on the `Manage access` option.
+
+  ![Openscale Tile](doc/source/images/services-wos-manageaccess.png)
+
+* To add users to the service instance, click the `Add users` button.
+
+  ![Openscale Tile](doc/source/images/services-wos-addusers.png)
+
+* For all of the user accounts, select the `Editor` role for each user and then click the `Add` button.
+
+  ![Openscale Tile](doc/source/images/services-wos-userrole.png)
+
+</details>
+
+### 4. Create COS bucket and get credentials
+
+* In your [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage)  instance, create a bucket with a globally unique name. The UI will let you know if there is a naming conflict. This will be used in cell *1.3.1* as *BUCKET_NAME*.
+
+* In your [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) instance, get the Service Credentials for use as `COS_API_KEY_ID`, `COS_RESOURCE_CRN`, and `COS_ENDPOINT`:
+
+  ![COS credentials](doc/source/images/cos-credentials.png)
+
+## 5. Create a notebook on Cloud Pak for Data
+
+* In your on-premise Cloud Pak for Data, click `New Project +` under Projects or, at the top of the page click `+ New` and choose the tile for `Data Science` and then `Create Project`.
```

Comment:
does this really require on-premise CPD?

Reply:
Yes, For these instructions. This is the supported notebook from the OpenScale team, so I want to use it. I don't think this pattern has had much use, so I don't think it's worth the effort to make it more universal.